tests/mapper/test_relational_node_mappers.py

```diff
@@ -0,0 +1,191 @@
+from stellargraph.core.graph import *
+from stellargraph.mapper.node_mappers import RelationalFullBatchNodeGenerator
+
+import networkx as nx
+import numpy as np
+import pytest
+import pandas as pd
+import scipy.sparse as sps
+
+
+def create_graph_features():
+    G = nx.MultiDiGraph()
+    G.add_nodes_from(["a", "b", "c"])
+    G.add_edges_from([("a", "b", "r1"), ("b", "c", "r1"), ("a", "c", "r2")])
+    return G, np.array([[1, 1], [1, 0], [0, 1]])
+
+
+class Test_RelationalFullBatchNodeGenerator:
+    """
+    Tests of FullBatchNodeGenerator class
+    """
+
+    n_feat = 2
+    target_dim = 5
+
+    gnx, features = create_graph_features()
+    nodes = list(gnx.nodes)
+    node_features = pd.DataFrame.from_dict(
+        {n: f for n, f in zip(nodes, features)}, orient="index"
+    )
+    G = StellarDiGraph(gnx, node_features=node_features)
```

Comment:
Please see my previous comments about the correct type of the graph input to RelationalFullBatchNodeGenerator. Does the input graph have to be directed or can it also be undirected? You only have a test for directed graphs.

Reply:
It can be both, I've added in unit tests for both directed and undirected